megamek/i18n/megamek/common/options/messages.properties

```diff
@@ -33,6 +33,10 @@ GameOptionsInfo.option.dumping_from_round.displayableName=first round for ammo d
 GameOptionsInfo.option.dumping_from_round.description=Number of the round that has to be at least reached before allowing ammo dumps.
 GameOptionsInfo.option.set_arty_player_homeedge.displayableName=Automatically set artillery home edge
 GameOptionsInfo.option.set_arty_player_homeedge.description=If checked, all of the players' artillery units will have their home edge set to the deployment edge of the player, NW and NE are North, SW and SE are South. \nUnchecked by default.
+GameOptionsInfo.option.set_default_team_1.displayableName=Default non-bot players to team 1
+GameOptionsInfo.option.set_default_team_1.description=Unchecked each player is assigned new team
```

Comment:
I feel this is not understandable unless you already know what it does. At least it should start with "When this option is unchecked, "